app/javascript/src/modules/proceedings.js

```diff
@@ -153,11 +151,18 @@ function showResults (results, inputText) {
       proceedingsContainer.insertBefore(element, proceedingsContainer.children[idx])
       // show hidden proceedings item
       show(element)
-      hide(document.querySelector('.no-proceeding-items'))
     })
-    // the below alerts screen reader users that results appeared on the page
-    const pluralizedMatches = pluralize(codes.length, 'match', 'matches')
-    ariaText = `${codes.length} ${pluralizedMatches} found for ${inputText}, use tab to move to options`
+    // once we have looped over all returned codes, check if any were shown to the user
+    // and set the appropriate ariaText and show/hide the no-proceedings found element
+    if (shown > 0) {
+      // the below alerts screen reader users that results appeared on the page
+      const pluralizedMatches = pluralize(codes.length, 'match', 'matches')
+      ariaText = `${codes.length} ${pluralizedMatches} found for ${inputText}, use tab to move to options`
+      hide(document.querySelector('.no-proceeding-items'))
+    } else {
+      show(document.querySelector('.no-proceeding-items'))
+      ariaText = `No results found matching ${inputText}`
```

Comment:
No biggee. I see this was repeated before in any event but would be nice to have one place where this is set/handled, even if just a method that is just called in the two places (here and line 167.

Reply:
Hmm, I can't see this actually being used 😕 
I might summon @naseberry for a consultation! 😄